tests/test_power_spectrum_and_bfactor.py

```diff
@@ -0,0 +1,72 @@
+import torch
+from cryo_challenge.power_spectrum_utils import _centered_ifftn, compute_power_spectrum
+from cryo_challenge._preprocessing.bfactor_normalize import (
+    _compute_bfactor_scaling,
+    bfactor_normalize_volumes,
+)
+
+
+def test_compute_power_spectrum():
```

Comment:
I think this works because the FT of a gaussian is radially symmetric. And so a the power of a ray from the origin (`[: box_size // 2, 0, 0]`) is the same as the power of the 3D object.

I'd put that note in the docstring to explain the logic of this text

Reply:
This just gave me an idea for a new test. I will make the change and implement the new test.